lib/collection/src/shards/transfer/mod.rs

```diff
@@ -31,10 +31,10 @@ const CONSENSUS_CONFIRM_TIMEOUT: Duration = defaults::CONSENSUS_META_OP_WAIT;
 #[derive(Debug, Clone, Hash, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
 pub struct ShardTransfer {
     pub shard_id: ShardId,
-    /// For resharding, a different target shard ID may be configured
-    /// By default the shard ID on the target peer is the same.
+    /// Target shard ID if different than source shard ID
+    ///
+    /// Used exclusively with `ReshardStreamRecords` transfer method.
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    #[schemars(skip)] // TODO(resharding): expose once we release resharding
     pub to_shard_id: Option<ShardId>,
```

Comment:
Is this actually used anywhere in the API? Telemetry? Or `JsonSchema` derived just in case?

Reply:
Still compiles without it, so I think it's safe to remove.